### ISSUE TITLE
Today and cancel buttons should be focusable with the keyboard

### ISSUE THREAD

**Issue by tomivirkki** [vaadin/vaadin-date-picker#277]
Today and cancel buttons should be focusable with the keyboard (part of the tab order when the overlay is open)

**Comment by platosha** [comment]
Fixed in #297
